chat-client/src/client/chat.ts

```diff
@@ -81,6 +82,17 @@ export const createChat = (
             case ERROR_MESSAGE:
                 mynahApi.showError((message as ErrorMessage).params)
                 break
+            case QUICK_ACTIONS_OPTIONS:
```

Comment:
As I mentioned in related PR https://github.com/aws/language-server-runtimes/pull/140/files#r1639904723, I think this should not be limited to Quick Actions only, and be made as more generic extendable object. This will allow to extend this mechanism in the future

Reply:
There is bunch on hard-coded string, that setup Chat panel messages specific to Amazon Q. Do we need to allow integrating clients to configure them at start time? Not sure if mechanism will be same though

Reply:
Made this more generic and named it CHAT_OPTIONS to allow for more configuration to be passed in the future